packages/fantasion-web/public/locales/cs/common.json

```diff
@@ -80,7 +82,7 @@
   "login-forgotten-password": "Zapomenuté heslo",
   "login-password": "Heslo",
   "login": "Přihlášení",
-  "login-register-general-info": "Zaregistrujte se a otevřete si možnost přihlašování na tábor. Můžete také přistupovat k aktualizacím z tábora.",
+  "login-register-general-info": "Bez registrace Vám bohužel nejsme schopní zajistit bezpečný přístup k Vašim přihláškám, objednávkám, fotkám a informacím z tábora. K jakýmkoliv dalším krokům je tedy nutná registrace emailem.",
```

Comment:
Ten novej text je napsanej strašně negativně, může to odradit potenciální zákazníky. Co bylo špatně na tom starém?

Reply:
Takhle by to mohlo být lepší